[WIP]Added umami

sudojarvis opened the pull request:
1)Added umami.conf file for umami configuration in nginx
2)Used [railway](https://railway.app/) for PostgreSQL database
3) we need to add .env file in nginx-tls folder.
`
DATABASE_URL=postgresql://${{ PGUSER }}:${{ PGPASSWORD }}@${{ PGHOST }}:${{ PGPORT }}/${{ PGDATABASE }}
DATABASE_TYPE=postgresql
HASH_SALT=generate_a_random_salt

POSTGRES_DB=railway
POSTGRES_USER=postgres
POSTGRES_PASSWORD=
`

@mwang87

sudojarvis committed:
added umami,used railway for PostgreSQL

mwang87 commented:
Just a few questions:

1. I think umami has its own db built into the docker container so wouldnt need a separate postgres
2. How do you actually hit the umami server via the reverse proxy? That is, what is the domain name we'd actually want to hit (if you could provide an example) because it doesnt specify which domain name to route

sudojarvis committed:
using umami database

sudojarvis committed:
added configuring using virtual-host

sudojarvis commented:
I have change the configuration in docker-compose ,added VIRTUAL_HOST rather than adding configraton file. Here is demo.
@mwang87 
https://user-images.githubusercontent.com/75030755/207975506-2b2bc4b8-d906-4550-a306-f4c1833cdbd7.mp4

EDIT: Also i am using umami's database and updated the docker-compose file for that.

mwang87 commented:
Thanks for the help, I took this code into antoher repository and it works!
